Mitrpolskiy_Alexey/src/component/header/header.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+
+import LoginModal from '../loginModal';
+import Menu from '../menu';
+
+import './header.css';
+
+const items = [
+	{ key: 'home', title: 'Home', href:'/' },
+	{ key: 'news', title: 'News', href:'/' },
+	{ key: 'blog', title: 'Blog', href:'/' }
+];
+
+import {
+	Collapse,
+	Navbar,
+	NavbarToggler,
+	NavbarBrand,
+	Nav,
+	NavItem,
+	NavLink,
+	UncontrolledDropdown,
+	DropdownToggle,
+	DropdownMenu,
+	DropdownItem } from 'reactstrap';
+
+export default class Header extends Component {
+	static defaultProps = {
+		size: 'big'
```

Comment:
```suggestion
		size: 'big',
```
Please use trailing comma :)